pkg/meshsync/sync.go

```diff
@@ -2,19 +2,19 @@ package meshsync
 
 import (
 	"context"
-	"meshery-operator/pkg/meshsync/models"
+	"github.com/layer5io/meshery-operator/pkg/meshsync/models"
 	"sync"
 )
 
 type Synchronizer interface {
 	Synchronize(context.Context, *sync.WaitGroup, <-chan struct{}) error
 }
 
-type Processor interface {
-	Process(context.Context, models.Event)
+type Fingerprinter interface {
```

Comment:
I guess I would have expected to see it's adjoining package imported as well. Perhaps, not necessary for now. 

```
import (
          "meshery-operator/pkg/meshsync/inkpad"
)
```